x-pack/plugins/enterprise_search/public/applications/app_search/components/app_search_gate/app_search_gate.tsx

```diff
@@ -0,0 +1,628 @@
+      }
+    ),
+  },
+  credentials: {
+    actionLabel: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.credentials.featureButtonLabel',
+      {
+        defaultMessage: 'Secure with Elasticsearch',
+      }
+    ),
+    actionLink:
+      'https://www.elastic.co/guide/en/elasticsearch/reference/current/document-level-security.html',
+    addOnLearnMoreLabel: undefined,
+    addOnLearnMoreUrl: undefined,
+    description: i18n.translate(
+      'xpack.enterpriseSearch.appSearch.gateForm.credentials.featureDescription',
+      {
+        defaultMessage:
+          'Manage your users and roles, and credentials for accessing your search endpoints',
+      }
+    ),
+    id: 'credentials',
+    learnMore: 'https://www.elastic.co/search-labs/blog/dls-internal-knowledge-search',
+    panelText: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.credentials.panelText', {
+      defaultMessage:
+        'Elasticsearch provides a comprehensive set of security features, including document-level security and role-based access control.',
+    }),
+    title: i18n.translate('xpack.enterpriseSearch.appSearch.gateForm.credentials.featureName', {
+      defaultMessage: 'Credentials and roles',
+    }),
+  },
+};
+
+interface FeatureOption {
+  id: string;
+  title: string;
+  description: string;
+  learnMore: string | undefined;
+  actionLabel: string;
+  actionLink: string;
+  panelText: string;
+  addOnLearnMoreLabel?: string;
+  addOnLearnMoreUrl?: string;
+}
+
+const getFeature = (id: string): FeatureOption | undefined => {
+  switch (id) {
+    case featuresList.webCrawler.id:
+      return featuresList.webCrawler;
+    case featuresList.analyticsAndLogs.id:
+      return featuresList.analyticsAndLogs;
+    case featuresList.synonyms.id:
+      return featuresList.synonyms;
+    case featuresList.relevanceTuning.id:
+      return featuresList.relevanceTuning;
+    case featuresList.curations.id:
+      return featuresList.curations;
+    case featuresList.searchManagementUis.id:
+      return featuresList.searchManagementUis;
+    case featuresList.credentials.id:
+      return featuresList.credentials;
+    default:
+      return undefined;
+  }
+};
+
+interface FeatureOptionsSelection {
+  dropdownDisplay: React.ReactNode;
+  inputDisplay: string;
+  value: string;
+}
+
+const getOptionsFeaturesList = (): FeatureOptionsSelection[] => {
+  const baseTranslatePrefix = 'xpack.enterpriseSearch.appSearch.gateForm.superSelect';
+
+  const featureList = Object.keys(featuresList).map((featureKey): FeatureOptionsSelection => {
+    const feature = getFeature(featureKey);
+    if (!feature) {
+      return {
+        dropdownDisplay: <></>,
+        inputDisplay: '',
+        value: '',
+      };
+    }
+
+    const featureTranslatePrefix = `${baseTranslatePrefix}.${feature.id}`;
+
+    return {
+      dropdownDisplay: (
+        <Fragment>
+          <strong>
+            {i18n.translate(`${featureTranslatePrefix}.title`, {
```

Comment:
This translate isn't needed since the `feature.title` values are all translated already.